src/web/timeline.rkt

```diff
@@ -178,6 +179,48 @@
 (define (average . values)
   (/ (apply + values) (length values)))
 
+(define (render-phase-mixed-sampling mixsample)
+  (define current-op '())
+  (define precisions '())
+  (define timings '())
+  (define final-list '())
+  
+  (for/list ([rec (in-list (sort mixsample string<? #:key first))])
+    (match-define (list operation precision milliseconds) rec)
+    (cond
+      [(equal? operation current-op)
+       (set! precisions (cons precision precisions))
+       (set! timings (cons milliseconds timings))]
+      [(equal? current-op '())
+       (set! current-op operation)
+       (set! precisions (cons precision precisions))
+       (set! timings (cons milliseconds timings))]
+      [else
+       (set! final-list (cons (list current-op precisions timings) final-list))
+       (set! current-op operation)
+       (set! precisions '())
+       (set! precisions (cons precision precisions))
+       (set! timings '())
+       (set! timings (cons milliseconds timings))]))
+  (set! final-list (cons (list current-op precisions timings) final-list))
+  
+  `((dt "Precisions")
+    (dd (details
+         (summary "Click to see histograms")
+         ,@(for/list ([rec (in-list final-list)])
+             (define n (random 100000))
```

Comment:
This is kinda dumb way, the id needs to be unique for every histogram, I made it like this